reddit.js

```diff
@@ -1,5 +1,30 @@
 var reddit = require("./data.json")
 
 // Write your solutions below
+// var titles = reddit.data.children.data.title;
+var titles = reddit.data.children;
+titles.forEach(function(title){
+	console.log(title.data.title);
+})
 
-console.log(reddit["data"]);
\ No newline at end of file
+var links = titles.map(function (link) {
```

Comment:
Similar to the above, 'link' may be a misleading parameter name. Since map is iterating over each child of the children array, 'child' or 'post' may be a better parameter name to use in this case.